dev-requirements.pip

```diff
@@ -0,0 +1,16 @@
+# Include base requirements
+-r requirements.pip
+
+# For unit tests
+coverage==4.2
+# GOTCHA: This is the last version we can build in Jenkins due to the setuptools version limit.
```

Comment:
I can investigate on fixing this, but I figured I would just wait for the new CI system first.